build/dependencies.props

```diff
@@ -207,6 +206,7 @@
     <SystemTextEncodingsWebPackageVersion>4.5.0</SystemTextEncodingsWebPackageVersion>
     <SystemThreadingChannelsPackageVersion>4.5.0</SystemThreadingChannelsPackageVersion>
     <SystemThreadingTasksDataflowPackageVersion>4.9.0</SystemThreadingTasksDataflowPackageVersion>
+    <SystemThreadingTasksExtensionsPackageVersion>4.5.4</SystemThreadingTasksExtensionsPackageVersion>
```

Comment:
Just curious but why did this jump from 4.5.2 to 4.5.4 -- has it been updated twice since we made it overridable in ProdCon?

Reply:
Yes that is the case.